include/linux/socket.h

```diff
@@ -207,7 +207,7 @@ struct ucred {
 				 * PF_SMC protocol family that
 				 * reuses AF_INET address family
 				 */
-#define AF_PSOCK	44	/* Xaptum router cards */
+#define AF_SCM	44	/* Xaptum router cards */
```

Comment:
Keep the changes to this file out of this PR, since it's not related to the setting up the USB interface.